app/javascript/components/application_form/preferences/AliceGriffithFields.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+import { map } from 'lodash'
+import { Select, Text } from 'react-form'
+
+import { buildFieldId } from './utils'
+import { typeOfProofValues } from './values'
+import Row from '~/components/atoms/Row'
+import Column from '~/components/atoms/Column'
+import FormGroup from '~/components/atoms/FormGroup'
+
+const buildTypeOfProofOptions = (values) => {
+  return map(values, (option) => (
+    { value: option, label: option }
+  ))
+}
+
+const AddressRow = ({fieldId}) => (
+  <React.Fragment>
+    <Row form>
+      <Column span={6} form>
+        <FormGroup>
+          <label htmlFor='address'>Alice Griffith Address</label>
```

Comment:
Nice--yay for better form accessibility. For this section, though, there is more than one address field on the page so `alice-griffith-address`, etc would probably be better, so as not to duplicate html ids.